service_mesh/service_mesh_install/ossm-jaeger.adoc

```diff
@@ -0,0 +1,30 @@
+[id="ossm-jaeger"]
+= Jaeger overview
```

Comment:
Should this title be "Understanding Jaeger" - to match with the table of content? Also "Jaeger overview" is a sub module aswell.